pages/1.10/storage/mount-disk-resources/index.md

```diff
@@ -46,10 +46,10 @@ In this example, a disk resource is added to a DC/OS agent post-install on a run
       ```
 
 4.  Clear agent state.
-    1. Remove volume mount discovery resource state with this command:
+    1. Cache volume mount discovery resource state with this command:
```

Comment:
Thanks for the contributing to the documentation. Do you think we can add a short sentence which explains why we need to cache this file? For example:
> Cache volume mount discovery resource state with the following command. DC/OS will check this file to generate a new resource configuration for the Mesos agent.

Reply:
I have added one line in `pages/1.12/storage/mount-disk-resources/index.md`. I have not backported it because the fix in DCOS_OSS-3921 has been backported to 1.10 and 1.11 but does not concern all 1.10 and 1.11 versions (but it will be in DC/OS 1.12.0).

Reply:
Hi, @ArmandGrillet. It makes sense to me. Please, ship it.